src/rdata/rfc1035/txt.rs

```diff
@@ -1035,6 +1016,28 @@ mod test {
         }
     }
 
+    #[test]
+    fn txt_strings_eq() {
+        let records = [["foo", "bar"], ["foob", "ar"]];
+
+        let records = records
+            .iter()
+            .map(|strings| {
+                let mut builder = TxtBuilder::<Vec<u8>>::new();
+                for string in strings {
+                    builder
+                        .append_charstr(
+                            CharStr::from_slice(string.as_bytes()).unwrap(),
+                        )
+                        .unwrap();
+                }
+                builder.finish().unwrap()
+            })
+            .collect::<Vec<_>>();
+
+        assert_ne!(records[0], records[1]);
```

Comment:
It's better to also include a test case the check two records are equal, i.e. `assert_eq!(...)`.

Reply:
Thank you, I included such a test now.